app/routes/pilots.js

```diff
@@ -13,13 +15,14 @@ router.get('/list', function (req, res, next) {
 
 /* GET specific pilot. */
 router.get('/:id', function (req, res, next) {
+    logger.debug('Router: /pilots/' + req.params.id);
     var pilot_id = req.params.id;
 
     Pilot.findById(pilot_id).populate('inventory.ware').exec(function (err, pilot) {
         if (err) return res.status(422).send('Problem loading the pilot:', error.message);
         if (!pilot) return res.status(404).send('Couldn\'t find the pilot');
 
-        console.log(pilot.inventory);
+        logger.info("" + pilot.inventory);
```

Comment:
This might not be the best way to do this. If you won't cast it to a string it prints the full javascript prototype.